GCDWebServer.podspec

```diff
@@ -35,7 +35,7 @@ Pod::Spec.new do |s|
   
   s.subspec "CocoaLumberjack" do |cs|
     cs.dependency 'GCDWebServer/Core'
```

Comment:
Why the change? Why can't it stay at 2.x? What will break if someone is using 2.x but now has to use 3.x because of this GCDWebServer update.